src/main/java/se/sundsvall/seabloader/api/InvoicesResource.java

```diff
@@ -0,0 +1,39 @@
+package se.sundsvall.seabloader.api;
+
+import static org.springframework.http.MediaType.ALL_VALUE;
+import static org.springframework.http.MediaType.APPLICATION_OCTET_STREAM_VALUE;
+import static org.springframework.http.MediaType.APPLICATION_PROBLEM_JSON_VALUE;
+import static org.springframework.http.MediaType.APPLICATION_XML_VALUE;
+
+import javax.validation.constraints.NotNull;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.zalando.problem.Problem;
+import org.zalando.problem.violations.ConstraintViolationProblem;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+
+@RestController
+@Validated
+@Tag(name = "Invoices", description = "Invoice resource")
+@RequestMapping("/invoices")
+public class InvoicesResource {
+
+	@PostMapping(consumes = { APPLICATION_OCTET_STREAM_VALUE, APPLICATION_XML_VALUE }, produces = APPLICATION_PROBLEM_JSON_VALUE)
+	@Operation(summary = "Create invoice")
+	@ApiResponse(responseCode = "204", description = "Successful operation", content = @Content(mediaType = ALL_VALUE, schema = @Schema(implementation = Void.class)))
+	@ApiResponse(responseCode = "400", description = "Bad request", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(oneOf = { Problem.class, ConstraintViolationProblem.class })))
+	@ApiResponse(responseCode = "500", description = "Internal Server error", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
+	public ResponseEntity<Void> createInvoice(@NotNull @RequestBody final byte[] body) {
+		return ResponseEntity.noContent().build();
```

Comment:
Question: should we have validation for correctness of incoming byte array? Maybe not part of this task, but I didn't see a task to add validation and wondered if it was left out on purpose or forgotten?